tests/e2e/testPages/prebid-main.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<html class="no-js">
+<head>
+	<meta charset="utf-8">
+  	<title>Prebid AppNexus</title>
+  	<meta name="description" content="">
+<script src="https://ajax.googleapis.com/ajax/libs/webfont/1.5.18/webfont.js"></script>
+<script>
+WebFont.load({
+    custom: {
+        families: ['Akkurat-Light', 'Akkurat-Regular', 'Akkurat-Bold'],
+        urls: ['//www.appnexus.com/static-assets/fonts/fonts.css']
+    }
+});
+</script>
+
+<script>
+	var pages = ['play-body',
```

Comment:
We need to change the name of the following test pages:
- 'play-body' should be 'prebid-body'
- 'play-body-ad-server' should be 'prebid-body-ad-server'